resource_mock/resources.py

```diff
@@ -23,6 +23,11 @@ def __init__(self,
                  ifus: Dict[str, Dict[str, str]]):
         self.fpu = fpu
         self.fpur = fpur
+        self.fpur[Site.GS].extend(['11013104','11013107','11020601','11022001',
+                                    '11023313','11023327','11023328','11023332',
+                                    '11023341','11023342','10000009','10005373',
+                                    '10005372','10005374','10005375','10005376',
+                                    '10005390'])
```

Comment:
This seems kind of hacky, no? Should this information be hard-coded? Just curious why it isn't getting passed in for `fpur` in the constructor.